packages/components/src/theme/detail.scss

```diff
@@ -144,6 +144,10 @@
   color: $brand-warning;
 }
 
+.detail__editing .hide-label {
```

Comment:
Rather than bringing this back (I realize it never left in the LKB case) I think we should try to make use of our formsy input, query-column aware, wrapping components. These inputs are ones like `TextInput` and `TextAreaInput` which wrap their corresponding formsy-react-components input but already provide support for labeling as we desire. This also better aligns `resolveDetailEditRenderer` and `QueryFormInputs` as the latter already makes use of these components. I've taken a first stab at what this looks like and it appears to work well.

I've taken the opportunity to see if we can slightly improve the handling of rendering labels on these inputs and it appears (after reading the `formsy-react-components` source code) that the supported pattern for not rendering a label is to set `layout="elementOnly"`. With that set the label rendering does fire off at all.

Take a look at the patch I've supplied to you and let me know what you think about this direction.

Reply:
The patch you provided looks good! Thanks. 
One small thing I noticed that we do lose by setting the `layout` to `elementOnly` is that the validations provided will show error styling in red and info below the input (for example the int field validation below):
<img width="1336" alt="Screen Shot 2022-11-18 at 2 59 24 PM" src="https://user-images.githubusercontent.com/6411206/202801647-c2700c2a-5171-445a-8402-0147a70d4766.png">

@labkey-nicka do you think that is something we want to retain / care about?

Reply:
Hmm, interesting (and sad). Does it work for us to remove the configuration of the layout and have it still respect `showLabel` without needing to hide it using CSS? ? If so, then I'd be fine with not configuring the layout on these components.

Reply:
Ah, I should mention that this was done in tandem with setting label to something like `{showLabel : this.renderLabel() : null}`. I'm fairly certain that will no longer work on its own. Probably better to let `FieldLabel` resolve itself to null in that case meaning just switch back to `label={this.renderLabel()}` for these components (besides `CheckboxInput` which as we discussed is unlike any of the others.

Reply:
Yup, had to revert back to `label={this.renderLabel()}` after removing the `layout={showLabel ? undefined : 'elementOnly'}`. But that brings us back to the `formsy-react-components` rendering the asterisk when the field is marked as required. 
<img width="1345" alt="Screen Shot 2022-11-21 at 9 56 51 AM" src="https://user-images.githubusercontent.com/6411206/203100780-4395a530-ab42-48f5-8218-74c3f352942a.png">

So unless you have another idea, I brought back the detail.scss for `hide-label`.

Reply:
😿